test/zspec/zcl/frame.test.ts

```diff
@@ -115,14 +130,35 @@ const GLOBAL_FRAME = Zcl.Frame.create(
     GLOBAL_HEADER.manufacturerCode,
     GLOBAL_HEADER.transactionSequenceNumber,
     GLOBAL_HEADER.commandIdentifier,
-    Zcl.Foundation.read.ID,
+    Zcl.Clusters.genBasic.ID,
```

Comment:
Why these changes (below too)? It doesn't match the variable names anymore (GLOBAL => Foundation).

Reply:
Because it is wrong, this should be the cluster ID, the command ID is in the header

Reply:
Whoops! Should have opened the full code 😛